pages/dkp/kommander/2.2/install/air-gapped/catalog/index.md

```diff
@@ -1,110 +1,254 @@
+
+This topic shows how to run Kommander on top of an [air-gapped Konvoy cluster][air-gap-konvoy] installation with catalog applications.
 
 ## Prerequisites
 
-- Install `git` in the local environment (from where a connection to air-gapped cluster is established).
+Before installing, ensure you have:
 
-## Host external repository on Gitea
+-   A Docker registry containing all the necessary Docker images, including the Kommander and [DKP catalog application][dkp_catalog_applications] images.
 
-To configure the Gitea server, follow these steps:
+-   A charts bundle file containing all Helm charts that Kommander installation needs, including the [DKP catalog applications][dkp_catalog_applications] charts.
 
-1.  Set the `VERSION` environment variable to the version of Kommander you want to install, for example:
+-   Connectivity with clusters attaching to the management cluster:
+    - Both management and attached clusters must connect to the Docker registry.
+    - Management cluster must connect to the attached cluster's API server.
+    - Management cluster must connect to load balancers created by some platform services.
 
-    ```bash
-    export VERSION=v2.2.0
-    ```
+-   A [configuration file][kommander-config] that you will adapt to your needs using the steps outlined in this topic. Make sure to create that file using the following command:
+
+  ```bash
+  kommander install --init --airgapped > install.yaml
+  ```
+
+-   All the prerequisites covered in [air-gapped Konvoy installation][air-gap-before-you-begin].
+
+-   [MetalLB enabled and configured][air-gap-install-metallb], which provides load-balancing services.
+
+### Kommander charts bundle
+
+The charts bundle is a gzipped Tar archive containing Helm charts, which are required during Kommander installation.
+Create the charts bundle with the Kommander CLI or downloaded along with the Kommander CLI.
+Execute this command to create the charts bundle:
+
+   ```bash
+   kommander create chart-bundle
+   ```
+
+Kommander creates `charts-bundle.tar.gz`.
+Optionally, specify the output using the `-o` parameter:
+
+   ```bash
+   kommander create chart-bundle -o [name of the output file]
+   ```
+
+### Kommander's internal Helm repository
+
+The Kommander charts bundle is pushed to Kommander's internal Helm repository.
+To inspect the contents:
+
+   ```bash
+   kommander get charts
+   ```
+
+Individual charts can be removed using:
+
+   ```bash
+   kommander delete chart [chartName] [chartVersion]
+   ```
+
+It is possible to push new charts as well:
+
+   ```bash
+   kommander push chart [chartTarball]
+   ```
+
+Or push a new bundle:
+
+   ```bash
+   kommander push chart-bundle [chartsTarball]
+   ```
+
+Check the built-in help text for each command for more information.
+
+### Use MetalLB
```

Comment:
I thought we were changing this in 2.2?   See https://jira.d2iq.com/browse/D2IQ-86454, etc.

Basically, because metal-lb *requires* per-cluster customization, and Kommander does not support that, metal-lb should not be a platform application; it should instead be deployed by Konvoy.  

I think we should remove or comment out this section for now while that ticket gets sorted out.

Reply:
This page is essentially a copy of the existing 2.2 installations instructions with the additional steps required for catalog applications (since that is an Enterprise only feature). I didn't actually change any of the install instructions, just added some additional steps. 

I'm not sure the status of that ticket/work but I would feel more comfortable leaving this here and let those changes happen when that ticket is worked if that's good with you.

Reply:
@sonali-mishra-22 what do you think ?

Reply:
I agree, we should handle removing or keeping metallb, in the bug ticket which is supposed to address that issue.